torch_xla/csrc/xla_lower_util.cpp

```diff
@@ -504,6 +504,25 @@ xla::XlaOp BuildDropout(xla::XlaOp input, float probability, xla::XlaOp seed) {
   return input * mask;
 }
 
+std::vector<xla::XlaOp> BuildNativeDropout(xla::XlaOp input, xla::XlaOp seed,
+                                           float probability,
+                                           c10::optional<bool> train) {
+  const xla::Shape& shape = ShapeHelper::ShapeOfXlaOp(input);
+  if (!train.has_value() || *train) {
+    xla::XlaOp prob =
+        XlaHelpers::ScalarBroadcast<float>(probability, shape, input.builder());
```

Comment:
here I think it should be `1 - probability`. Because in the `BuildBernoulli` call. It will return a mask that `prob` of the elements are 1, the rest are 0.